firefox-ios/firefox-ios-tests/Tests/XCUITests/TodayWidgetTests.swift

```diff
@@ -323,6 +323,9 @@ class TodayWidgetTests: BaseTestCase {
     // TESTS
     // https://mozilla.testrail.io/index.php?/cases/view/2769289
     func testNewSearchWidget() throws {
+        if #unavailable(iOS 16) {
```

Comment:
@mdotb-moz Sorry. I can't get the tests to pass on iOS 15. How about on your end?